Tests/Marketplace/marketplace_services.py

```diff
@@ -701,35 +759,38 @@ def prepare_release_notes(self, index_folder_path, build_number):
                     latest_release_notes = found_versions[0].vstring
 
                     print_color(f"Latest ReleaseNotes version is: {latest_release_notes}", LOG_COLORS.GREEN)
+                    # load latest release notes
+                    latest_rn_file = latest_release_notes.replace('.', '_')
+                    latest_rn_path = os.path.join(release_notes_dir, latest_rn_file + '.md')
+
+                    with open(latest_rn_path, 'r') as changelog_md:
+                        release_notes_lines = changelog_md.read()
+                    release_notes_lines = self._clean_release_notes(release_notes_lines)
+
                     if self._current_version != latest_release_notes:
                         # TODO Need to implement support for pre-release versions
                         print_error(f"Version mismatch detected between current version: {self._current_version} "
                                     f"and latest release notes version: {latest_release_notes}")
                         task_status = False
-                        return task_status
+                        return task_status, not_updated_build
                     else:
                         with open(changelog_index_path, "r") as changelog_file:
                             changelog = json.load(changelog_file)
 
                             if latest_release_notes in changelog:
-                                changelog[latest_release_notes][  # update latest release notes build numbers
-                                    'displayName'] = f'{latest_release_notes} - {build_number}'
+                                changelog[latest_release_notes][  # update latest release notes
+                                    'displayName'] = f'{latest_release_notes} - R{build_number}'
+                                changelog[latest_release_notes]['releaseNotes'] = release_notes_lines
```

Comment:
Should we update also `released` time here (in case of override)?

Reply:
@IkaDemisto yes